barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.cpp

```diff
@@ -1081,17 +1097,8 @@ void UltraCircuitBuilder_<Arithmetization>::create_sort_constraint_with_edges(
     // TODO(https://github.com/AztecProtocol/barretenberg/issues/879): This was formerly a single arithmetic gate. A
     // dummy gate has been added to allow the previous gate to access the required wire data via shifts, allowing the
     // arithmetic gate to occur out of sequence.
-    create_dummy_gate(
-        blocks.main, variable_index[variable_index.size() - 1], this->zero_idx, this->zero_idx, this->zero_idx);
-    create_big_add_gate({ variable_index[variable_index.size() - 1],
-                          this->zero_idx,
-                          this->zero_idx,
-                          this->zero_idx,
-                          1,
-                          0,
-                          0,
-                          0,
-                          -end });
+    create_dummy_gate(block, variable_index[variable_index.size() - 1], this->zero_idx, this->zero_idx, this->zero_idx);
+    create_add_gate({ variable_index[variable_index.size() - 1], this->zero_idx, this->zero_idx, 1, 0, 0, -end });
```

Comment:
using `create_add_gate` instead since there was no need for the "big add gate" functionality